start_pokealarm.py

```diff
@@ -97,6 +97,8 @@ def check_for_update():
     masterfile_vreq = "https://api.github.com/repos/WatWowMap/Masterfile-" \
         "Generator/commits?path=master-latest-everything.json&per_page=1"
     pogoapi_vreq = "https://pogoapi.net/api/v1/api_hashes.json"
+    shiny_possible_vreq = "https://api.github.com/repos/jms412/PkmnShinyMap/" \
```

Comment:
I think this url might be better:
`https://raw.githubusercontent.com/jms412/PkmnShinyMap/main/shinyPossible.json`

Reply:
This first url is to make a request to the github api to obtain the last commit signature and know if PA need to update the local file.